subscription-engine/kafka-ws/examples/docker-compose.yml

```diff
@@ -84,9 +84,9 @@ services:
     # volumes:
     #  - ./certs/server/:/certs/:Z
     # to dev
-    volumes:
-     - ../app/:/opt/kafka-ws/app/:Z # need to run before 'npm run parser:compile'
-    command: ["npm", "run", "dev"]
+    # volumes:
```

Comment:
Could be removed?

Reply:
I can remove. But my idea was to leave it, if someone is developing it just need to uncomment this.
What do you think?